test/body.js

```diff
@@ -0,0 +1,53 @@
+const tape = require('tape')
+const turbo = require('../')
+const http = require('http')
+
+tape('test http head response response has no body', function (t) {
+  const server = turbo.createServer(function (req, res) {
+    res.setHeader('Status', 200)
+    res.write('FAIL')
+  })
+
+  server.listen(0, function () {
+    const req = http.request({
+      port: server.address().port,
+      method: 'HEAD',
+      path: '/'
+    }, function (res) {
+      res.on('end', function () {
+        server.close()
+      })
+      res.resume()
+    })
+    req.on('error', function (e) {
+      t.pass('should throw parse error.')
+      t.end()
+    })
+    req.end()
+  })
+})
+
+tape('test http head response when req has no body', function (t) {
+  const server = turbo.createServer(function (req, res) {
+    res.setHeader('Status', 200)
```

Comment:
did you mean res.statusCode = 200 ?

Reply:
@mafintosh yes, but i see it's by default here - https://github.com/mafintosh/turbo-http/blob/master/lib/response.js#L28 and could just do `res.statusCode = 200`. i'll remove it